.mailmap

```diff
@@ -4,12 +4,27 @@ Brian Gianforcaro <bgianf@serenityos.org> <b.gianfo@gmail.com>
 <gbeutner@serenityos.org> <gunnar@beutner.name>
 Max Wipfli <max.wipfli@serenityos.org> <mail@maxwipfli.ch>
 Sam Atkins <atkinssj@serenityos.org> <atkinssj@gmail.com>
+<atkinssj@serenityos.org> <github@samatkins.co.uk>
 Karol Kosek <krkk@serenityos.org> <krkk@krkk.ct8.pl>
 davidot <davidot@serenityos.org> <david.tuin@gmail.com>
 Tobias Christiansen <tobyase@serenityos.org> <tobi@tobyase.de>
 kleines Filmröllchen <filmroellchen@serenityos.org> <malu.bertsch@gmail.com>
+<filmroellchen@serenityos.org> <28656157+kleinesfilmroellchen@users.noreply.github.com>
 Lenny Maiorani <lenny@serenityos.org> <lenny@colorado.edu>
 Kenneth Myhra <kennethmyhra@serenityos.org> <kennethmyhra@gmail.com>
 networkException <networkexception@serenityos.org> <git@nwex.de>
 networkException <networkexception@serenityos.org> <github@nwex.de>
 Thomas Voss <mail@thomasvoss.com> <thomasvoss@live.com>
+<kling@serenityos.org> <awesomekling@gmail.com>
+<kling@serenityos.org> <ankl@hms.se>
+<liavalb@gmail.com> <51659396+supercomputer7@users.noreply.github.com>
+Liav A <liavalb@gmail.com>
+<akaster@serenityos.org> <andrewdkaster@gmail.com>
+<idan.horowitz@gmail.com> <idan.horowitz@ibm.com>
+<leon.a@serenityos.org> <37629766+Hendiadyoin1@users.noreply.github.com>
+Hendiadyoin1 <leon.a@serenityos.org> <leon2002.la@gmail.com>
+Jesse Buhagiar <jooster669@gmail.com> <Jooster669@gmail.com>
+Jesse Buhagiar <jooster669@gmail.com> <jesse.buhagiar@student.rmit.edu.au>
```

Comment:
```suggestion
<jooster669@gmail.com> <jesse.buhagiar@student.rmit.edu.au>
```

Reply:
Without specifying the name explicitly we end up with this - so I'm going to keep that as is:

```
   8 Author: Jesse <jooster669@gmail.com>
 146 Author: Jesse Buhagiar <jooster669@gmail.com>
```